apps/meteor/tests/end-to-end/api/livechat/20-messages.ts

```diff
@@ -51,5 +52,16 @@ describe('LIVECHAT - messages', () => {
 				expect(quotedMessageAttachments).to.have.property('text').that.is.equal(agentMsgSentence);
 			}
 		});
+
+		it('should verify if visitor is receiving a message with a image attachment', async () => {
+			const {
+				room: { _id: roomId },
+				visitor: { token },
+			} = await startANewLivechatRoomAndTakeIt();
+
+			const imgMessage = await uploadFile(roomId, token);
+
+			expect(imgMessage).to.have.property('files').that.is.an('array');
```

Comment:
Is there something else we can check on here? Like, the array including the actual file you sent and not just being an empty array 👀